File: ePrijevozSarajevo/ePrijevozSarajevo.Model/Delay.cs
namespace ePrijevozSarajevo.Model
{
    public class Delay
    {
        public int DelayId { get; set; }
        public string Reason { get; set; } = null!;
        public Route? Route { get; set; } 
        public int? RouteId { get; set; }
        public int? DelayAmountMinutes { get; set; }
        public Type? Type { get; set; }
        public int? TypeId { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public int? CurrentUserId { get; set; }
    }   
}

File: ePrijevozSarajevo/UI/eprijevoz_desktop/lib/models/delay.dart
import 'package:json_annotation/json_annotation.dart';

part 'delay.g.dart';

@JsonSerializable()
class Delay {
  int? delayId;
  String? reason;
  int? routeId;
  int? delayAmountMinutes;
  int? typeId;

  Delay(
      {this.delayId,
      this.reason,
      this.routeId,
      this.delayAmountMinutes,
      this.typeId});

  factory Delay.fromJson(Map<String, dynamic> json) => _$DelayFromJson(json);

  Map<String, dynamic> toJson() => _$DelayToJson(this);
}
